conf/app.conf

```diff
@@ -143,7 +146,7 @@ log.error.output = stderr
 #  [dev] section for documentation of the various settings
 [prod]
 mode.dev = false
-
+server.host=${CC_SERVER_HOST}
```

Comment:
server.host is a variable that I created, because we can only retrieve the addres using revel.Server.Addr and how you can see, this get a wrong value, in production get 127.0.0.1!!!

So... the flow in production is:

environment CC_SERVER_HOST -> app.conf sever.host -> used to build the URL 

`CC_SERVER_HOST=http://calculator.coolclimatenetwork.net/`

In Dev the flux is the same, but since we don't set the server.host don't take the variable